README.md

```diff
@@ -0,0 +1,80 @@
+# REST API
+
+This rest api provides necessay backend support functionality create, read, update and delete. Currently it is saving data on the object called storage which provides a vary little persistance during server session.
+
+### Dependencies:
+
+- http
+- url
+- querystring
+- node-uuid
+- superagent
+- mocha
+- chai
+
+This API is structured on a Model View Controller(MVC) architecture pattern. The base technologies are node.js server, node.http module.
+
+
+POST:
+example: http POST localhost:3000/api/schemaname
+data: name='some name' content='some content'.
+
+result:
+
+~~~
```

Comment:
Wow, I didn't know you could use `~~~` for code blocks in markdown. You should switch to using triple `, since that is the norm.